Makefile

```diff
@@ -156,7 +156,9 @@ dev_wrapper:
 
 test_base:
 ifneq ($(findstring docker,$(CONTAINER_CMD)),)
+    ifeq ($(DEV),n)
 	$(MAKE) build
+    endif
```

Comment:
Is this change related?

Reply:
it's not, I was hoping to get this into master because the DEV=y functionality needs this small tweak